src/blocklet/index.md

```diff
@@ -1,16 +1,18 @@
 ---
-title: Blocklet
-description: 'Blocklet'
-keywords: 'abtnode,blocklet'
+title: Blocklets
+description: 'Blocklets'
+keywords: 'abtnode, blocklets'
 author: wangshijun
 category: ''
 layout: documentation
 tags:
   - abtnode
 ---
 
-This chapter will introduce Blocklet related content,include:
+Previously, you have deployed the ABT Node and have installed Blocklets from the [Marketplace](../marketplace). In this section, you will work with Blocklet administration features available in ABT Node. You will also look at the various types of Blocklets available with ABT Node.
 
-- [Basic Operations](./basic-operations)
-- [Blocklet Environments](./environments)
+## Contents
+
+- [Blocklet Management](./basic-operations)
+- [Configure Blocklet](./environments)
 - [Core Blocklets](./core-blocklets)
```

Comment:
The core blocklet page needs an update, since some blocklets are deprecated.

Reply:
May I know how to determine core blocklets as there are many on the market place?

Reply:
Core blocklet can be found https://registry.arcblock.io/

- Block explorer: the universal block explorer that supports query/browse/display data from any blockchain
- ABT Node Launcher: Launch node or service, private beta is not there yet, but should be included
- Certificate Manager: automatically take over SSL certificate generating and renewing within abt node
- FS Chain Manager: get a development chain up and running instantely
- Wallet Playground: show the potential of arcblock platform with demos
- NFT Marketplace: decentralized marketplace for buy and sell NFT
- NFT Store: start your NFT online business with a few clicks

Reply:
Updated the list of core blocklets.